mapie/tests/test_multi_label_classification.py

```diff
@@ -0,0 +1,599 @@
+def test_results_for_same_alpha(strategy: str) -> None:
+    """
+    Test that predictions and intervals
+    are similar with two equal values of alpha.
+    """
+    args = STRATEGIES[strategy][0]
+    mapie_clf = MapieMultiLabelClassifier()
+    mapie_clf.fit(X, y)
+    _, y_ps = mapie_clf.predict(
+        X,
+        alpha=[0.1, 0.1],
+        method=args["method"],
+        bound=args["bound"],
+        delta=.1
+    )
+    np.testing.assert_allclose(y_ps[:, 0, 0], y_ps[:, 0, 1])
+    np.testing.assert_allclose(y_ps[:, 1, 0], y_ps[:, 1, 1])
+
+
+@pytest.mark.parametrize("strategy", [*STRATEGIES])
+def test_results_for_partial_fit(strategy: str) -> None:
+    """
+    Test that predictions and intervals
+    are similar with two equal values of alpha.
+    """
+    args = STRATEGIES[strategy][0]
+    clf = MultiOutputClassifier(LogisticRegression()).fit(X, y)
+    mapie_clf = MapieMultiLabelClassifier(clf)
+    mapie_clf.fit(X, y)
+
+    mapie_clf_partial = MapieMultiLabelClassifier(clf)
+    for i in range(len(X)):
+        mapie_clf_partial.partial_fit(
+            X[i][np.newaxis, :],
+            y[i][np.newaxis, :]
+        )
+
+    y_pred, y_ps = mapie_clf.predict(
+        X,
+        alpha=[0.1, 0.1],
+        method=args["method"],
+        bound=args["bound"],
+        delta=.1
+    )
+
+    y_pred_partial, y_ps_partial = mapie_clf_partial.predict(
+        X,
+        alpha=[0.1, 0.1],
+        method=args["method"],
+        bound=args["bound"],
+        delta=.1
+    )
+    np.testing.assert_allclose(y_pred, y_pred_partial)
+    np.testing.assert_allclose(y_ps, y_ps_partial)
+
+
+@pytest.mark.parametrize("strategy", [*STRATEGIES])
+@pytest.mark.parametrize(
+    "alpha", [np.array([0.05, 0.1]), [0.05, 0.1], (0.05, 0.1)]
+)
+def test_results_for_alpha_as_float_and_arraylike(
+    strategy: str, alpha: Any
+) -> None:
+    """Test that output values do not depend on type of alpha."""
+    args = STRATEGIES[strategy][0]
+    mapie_clf = MapieMultiLabelClassifier()
+    mapie_clf.fit(X, y)
+    y_pred_float1, y_ps_float1 = mapie_clf.predict(
+        X,
+        alpha=alpha[0],
+        method=args["method"],
+        bound=args["bound"],
+        delta=.9
+    )
+    y_pred_float2, y_ps_float2 = mapie_clf.predict(
+        X,
+        alpha=alpha[1],
+        method=args["method"],
+        bound=args["bound"],
+        delta=.9
+    )
+    y_pred_array, y_ps_array = mapie_clf.predict(
+        X,
+        alpha=alpha,
+        method=args["method"],
+        bound=args["bound"],
+        delta=.9
+    )
+    np.testing.assert_allclose(y_pred_float1, y_pred_array)
+    np.testing.assert_allclose(y_pred_float2, y_pred_array)
+    np.testing.assert_allclose(y_ps_float1[:, :, 0], y_ps_array[:, :, 0])
+    np.testing.assert_allclose(y_ps_float2[:, :, 0], y_ps_array[:, :, 1])
+
+
+@pytest.mark.parametrize("strategy", [*STRATEGIES])
+def test_results_single_and_multi_jobs(strategy: str) -> None:
```

Comment:
Parallel processing is disabled
It is a test for future?

Reply:
yes